include/deal.II/opencascade/utilities.h

```diff
@@ -274,9 +274,9 @@ namespace OpenCASCADE
   /**
    * Given a TopoDS_Face @p face and the reference coordinates within this
    * face, returns the corresponding point in real space, the normal to the
-   * surface at that point and the mean curvature as a tuple.
+   * surface at that point and the min and max curvatures as a tuple.
    */
-  std_cxx11::tuple<Point<3>, Point<3>, double >
+  std_cxx11::tuple<Point<3>, Point<3>, double, double>
```

Comment:
Is it the min and max over the entire face? Maybe add this here and in the other place you modified. 

While you are there: a normal vector is really the difference between two points, which we usually represent using `Tensor <1,dim>`. Would you mind making this change to the second template argument here and below?